test/controllers/task_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class TaskControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
These empty tests can be deleted. You may want check out the TDD on Rails workshop on upcase to learn how to do testing.

Reply:
Cool, I'll delete these for now and work on adding some tests after this PR.